docs/data/joy/getting-started/templates/email/components/EmailContent.tsx

```diff
@@ -54,24 +71,105 @@ export default function EmailContent() {
+            color="success"
+            open={open[0]}
+            onClose={() => handleSnackbarClose(0)}
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+            startDecorator={<CheckCircleRoundedIcon />}
+            endDecorator={
+              <Button
+                onClick={() => handleSnackbarClose(0)}
+                size="sm"
+                variant="soft"
+                color="neutral"
+              >
+                Dismiss
+              </Button>
+            }
+          >
+            Your message has been sent.
+          </Snackbar>
+          <Button
+            size="sm"
+            variant="plain"
+            color="neutral"
+            startDecorator={<ForwardToInboxRoundedIcon />}
+            onClick={() => handleSnackbarOpen(1)}
+          >
+            Forward
+          </Button>
+          <Snackbar
+            color="success"
+            open={open[1]}
+            onClose={() => handleSnackbarClose(1)}
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+            startDecorator={<CheckCircleRoundedIcon />}
+            endDecorator={
+              <Button
+                onClick={() => handleSnackbarClose(1)}
+                size="sm"
+                variant="soft"
+                color="neutral"
+              >
+                Dismiss
+              </Button>
+            }
+          >
+            Your message has been forwarded.
+          </Snackbar>
+          <Button
+            size="sm"
+            variant="plain"
+            color="danger"
+            startDecorator={<DeleteRoundedIcon />}
+            onClick={() => handleSnackbarOpen(2)}
+          >
+            Delete
+          </Button>
+          <Snackbar
+            color="danger"
+            open={open[2]}
+            onClose={() => handleSnackbarClose(2)}
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+            startDecorator={<CheckCircleRoundedIcon />}
+            endDecorator={
+              <Button
+                onClick={() => handleSnackbarClose(2)}
+                size="sm"
+                variant="soft"
+                color="neutral"
+              >
+                Dismiss
+              </Button>
+            }
+          >
+            Your message has been deleted.
+          </Snackbar>
         </Box>
       </Box>
       <Divider sx={{ mt: 2 }} />
       <Box
         sx={{ py: 2, display: 'flex', flexDirection: 'column', alignItems: 'start' }}
       >
-        <Typography level="title-lg" textColor="text.primary">
-          Details for our Yosemite Park hike
-        </Typography>
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+          <Typography level="title-lg" textColor="text.primary">
+            Details for our Yosemite Park hike
+          </Typography>
+          <Chip size="sm" variant="outlined" color="warning">
+            Personal
+          </Chip>
+        </Box>
```

Comment:
```suggestion
        <Typography
          level="title-lg"
          textColor="text.primary"
          endDecorator={
            <Chip size="sm" variant="outlined" color="warning">
              Personal
            </Chip>
          }
        >
          Details for our Yosemite Park hike
        </Typography>
```

Using `endDecorator` can help with the layout because you don't need a wrapper with flexbox.